Sample/app/build.gradle

```diff
@@ -26,7 +29,7 @@ dependencies {
     // implementation project(':lib')
 
     // Use the following line to include client library for Face API from Maven Central Repository
-    implementation 'com.microsoft.projectoxford:face:1.4.3'
+    //implementation 'com.microsoft.projectoxford:face:1.4.3'
```

Comment:
We will first release a new version of Android SDK and then let's update this to use the public one.

Reply:
Not sure about this... the sample should then use the local module, isn't it?

Reply:
It should use the publicly released Android SDK on maven. We will first upgrade the SDK to 1.4.4 when the code review is complete, and then let you know, then pls help change this to refer to the 1.4.4. Does it make sense to you?

Reply:
ok